crypto/ed25519.go

```diff
@@ -51,7 +51,8 @@ func ParseAddress(hrp, saddr string) (PublicKey, error) {
 	if phrp != hrp {
 		return EmptyPublicKey, ErrIncorrectHrp
 	}
-	if len(paddr) != 33 /* compressed public key size */ {
+	const CompressedPublicKeySize = 33
```

Comment:
You should define this at the top of the file instead of inline (so we only need to define it once), not each call of `ParseAddress`.

Reply:
I thought of that, I don't know why I left it there. Thanks.